_posts/minutes/2016-09-16-minutes.markdown

```diff
@@ -0,0 +1,93 @@
+#### BAKERSFIELD COLLEGE
+
+#### Computer Science Club
+
+#### 1801 Panorama Drive, Business Building, Room 5
+
+#### Bakersfield, California 93305
```

Comment:
This top part might be better suited for the minutes template, if it will be the same each time.